src/hermes_claude_agent_sdk/parity/catalog.py

```
@@ -0,0 +1,376 @@
+    consecutive = policy.get("consecutive_passes")
+    if type(minimum) is not int or minimum < 1:
+        raise CatalogViolation(f"{field}.minimum_passes must be a positive integer")
+    if type(consecutive) is not int or consecutive < 1:
+        raise CatalogViolation(f"{field}.consecutive_passes must be a positive integer")
+    if consecutive < minimum:
+        raise CatalogViolation(f"{field}.consecutive_passes cannot be below minimum_passes")
+    triggers = frozenset(_string_list(policy.get("triggers", []), f"{field}.triggers", allow_empty=True))
+    if not triggers <= _REPEAT_TRIGGERS:
+        raise CatalogViolation(
+            f"{field}.triggers contains unsupported values: {sorted(triggers - _REPEAT_TRIGGERS)}"
+        )
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)